### PR TITLE
test(e2e): fix e2e tests

### PR DESCRIPTION
**Pull request by puru-hk**
Fix datepicker e2e test cases

## Related Issue(s)

closes none

## Description

fixes broken e2e on daily build
adds missing path to upload-collection (core) e2e test to config file
extends saucelabs test suite max time (for when slow connection/performance affects tests)

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [n/a] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

**Review by N1XUS — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by dmitry-stepanenko — APPROVED** [review]

File: e2e/wdio/platform/fixtures/testData/date-time-picker.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const date = `${new Date().getMonth() + 1}/1/2021, 1:01 PM`;
+export const date = `${new Date().getMonth() + 1}/1/2022, 1:01 PM`;
```
Comment (dimamarksman) [inline e2e/wdio/platform/fixtures/testData/date-time-picker.ts:1]: why `2022` and not `new Date().getFullYear()`?

**Comment by dmitry-stepanenko** [comment]
@puru-hk I've pushed into your branch in order to get this merged ASAP

File: e2e/wdio/platform/fixtures/testData/date-picker.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 export const date3 = `${currentMonth}/1/${currentYear} - ${currentMonth}/15/${currentYear}`;
-export const date4 = `0${currentMonth}/01/22 // 0${currentMonth}/15/22`;
+export const date4 = `0${currentMonth}/01/${currentYearShort} // 0${currentMonth}/15/${currentYearShort}`;
 export const date5 = `${currentMonth}/1/${currentYear}`;
 export const date6 = `1.0${currentMonth}.${currentYear} г.`;
 export const date7 = `01/0${currentMonth}/${currentYear}`;
 export const date8 = `1.${currentMonth}.${currentYear}`;
 export const date9 = '1/2/1990';
-export const date10 = `0${currentMonth}/01/22 // 0${currentMonth}/15/22`;
+export const date10 = `0${currentMonth}/01/${currentYearShort} // 0${currentMonth}/15/${currentYearShort}`;
```
Comment (dimamarksman) [inline e2e/wdio/platform/fixtures/testData/date-picker.ts:14]: looks like `0` month appending will start failing in 9 months :)

Comment (dimamarksman) [reply e2e/wdio/platform/fixtures/testData/date-picker.ts:14]: I wondered how this worked previously

**Comment by dimamarksman** [comment]
created a separate issue to keep this kind of test simpler https://github.com/SAP/fundamental-ngx/issues/7505

**Review by dimamarksman — APPROVED** [review]

**Comment by dimamarksman** [comment]
@puru-hk ` e2e_smoke (suite coreB)` is failing

**Pull request merged**